src/db/mormot.db.raw.oracle.pas

```diff
@@ -1693,6 +1695,10 @@ constructor TSqlDBOracleLib.Create(LibraryFileName: TFileName);
   SupportsInt64Params := (major_version > 11) or
                          ((major_version = 11) and
                           (minor_version > 1));
+  if major_version > 7 then

```

Comment:
Why do we need to change according to the version?

Reply:
Change according to the version is not a very important thing. Newer Oracle versions accepts longer varchar param so code for newer version will work also for older Oracle version. At most more memory will be allocated for buffers, but this should not have negative consequences, because memory will be released just after SQL execution.